docs/api-guide.md

```diff
@@ -654,14 +654,6 @@ You can't specify clustering-key boundaries and orderings in `Scan` without spec
 
 `Put` is an operation to put a record specified by a primary key. The operation behaves as an upsert operation for a record, in which the operation updates the record if the record exists or inserts the record if the record does not exist.
 
-{% capture notice--info %}
-**Note**
-
-When you update an existing record, you need to read the record by using `Get` or `Scan` before using a `Put` operation.
-{% endcapture %}
-
-<div class="notice--info">{{ notice--info | markdownify }}</div>
-
```

Comment:
Just to confirm, do we need to remove this note from this doc on the version branches?

Reply:
No, the implicit pre-read feature will be supported starting from version `3.11`, so the note is still required for version `3.10` and earlier versions.

Reply:
Understood. Thank you!